<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestockPerishable</class>
 <widget class="QWidget" name="RestockPerishable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Restock Perishable Product</string>
  </property>
  <widget class="QLabel" name="label_Heading">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Restock Perishable Product</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>281</width>
     <height>92</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="50,100,50">
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Stock"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="error_Stock">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Stock">
      <property name="text">
       <string>Stock Added</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Barcode">
      <property name="text">
       <string>Barcode</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Barcode"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="error_Barcode">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Expiry">
      <property name="text">
       <string>Expiry Date</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_Expiry">
      <property name="placeholderText">
       <string>yyyy-mm-dd</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="error_Expiry">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>182</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QPushButton" name="restock">
      <property name="text">
       <string>Restock</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>RestockPerishable</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
